packaging/windows/split.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+const chunkSize = 1024 * 1024 * 1024 // 1GiB chunk size
+
+func main() {
+	if len(os.Args) != 2 {
+		log.Fatal("Split takes only one argument (the file to split)")
+	}
+
+	if err := split(os.Args[1]); err != nil {
+		log.Fatal(err.Error())
+	}
+}
+
+func split(filePath string) error {
+	bundle, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+	defer bundle.Close()
+
+	bundleFileStat, err := bundle.Stat()
+	if err != nil {
```

Comment:
`bundleName := filepath.Base(filePath)`, and you can get rid of the `bundle.Stat()` call, and the `bundleFileStat.Name()` calls.